en_test.py

```diff
@@ -3,6 +3,8 @@
 import random
 import json
 
+__author__ = "dsm_helper"
```

Comment:
__author__ is better? right?  
Add single comment

Reply:
Then, Is there anything else good words without "author"?

Reply:
No, You were right 👍

Reply:
thx :D